.github/workflows/drift-check.yml

```diff
@@ -33,7 +33,15 @@ jobs:
           workload_identity_provider: ${{ secrets.WORKLOAD_IDENTITY_PROVIDER }}
           service_account: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
 
-      - uses: simenandre/setup-gke-gcloud-auth-plugin@v1
+      - uses: 'google-github-actions/setup-gcloud@v2'
+        with:
+          install_components: 'gke-gcloud-auth-plugin'
+
+      - name: Get GKE credentials
+        uses: 'google-github-actions/get-gke-credentials@v2'
+        with:
+          cluster_name: core-cluster
+          location: europe-north1
```

Comment:
Don't think we need this, we are not using kubectl for anything (Pulumi handles this step itself)